Description: Use /etc for ntp.keys
 The default path for the ntp.keys file is in /usr/local/etc.  We use /etc
 instead.  This involves updating some paths, and then removing documentation
 which is moot in light of that change.
Forwarded: not-needed
Origin: vendor
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2016-12-17
--- a/docs/includes/ntpkeygen-body.txt
+++ b/docs/includes/ntpkeygen-body.txt
@@ -34,12 +34,7 @@
 
 The safest way to run the +ntpkeygen+ program is logged in directly as
 root. The recommended procedure is change to the keys directory, usually
-+/usr/local/etc+, then run the program.
-
-Installing the keys as root might not work in NFS-mounted shared file
-systems, as NFS clients may not be able to write to the shared keys
-directory, even as root. In this case, NFS clients can specify the files
-in another directory such as +/etc+ using the _keysdir_ command.
++/etc+, then run the program.
 
 [[access]]
 == Key file access and location ==
@@ -62,8 +57,7 @@
 command.
 
 All files are installed by default in the keys directory
-+/usr/local/etc+, which is normally in a shared filesystem in
-NFS-mounted networks. The actual location of the keys directory and each
++/etc+. The actual location of the keys directory and each
 file can be overridden by configuration commands, but this is not
 recommended. Normally, the files for each host are generated by that
 host and used only by that host, although exceptions exist as noted
--- a/wafhelpers/configure.py
+++ b/wafhelpers/configure.py
@@ -275,7 +275,7 @@
         for header, sizeof in sorted(sizeofs, key=lambda x: x[1:]):
                 check_sizeof(ctx, header, sizeof)
 
-        ctx.define("NTP_KEYSDIR", "%s/etc" % ctx.env.PREFIX, comment="NTP key file directory")
+        ctx.define("NTP_KEYSDIR", "/etc", comment="NTP key file directory")
         ctx.define("GETSOCKNAME_SOCKLEN_TYPE", "socklen_t", quote=False, comment="socklen type")
         ctx.define("DFLT_RLIMIT_STACK", 50, comment="Default stack size")
 
--- a/docs/includes/auth-commands.txt
+++ b/docs/includes/auth-commands.txt
@@ -15,7 +15,7 @@
   are used as described above. Unless the complete path and name of the
   file are specified, the location of a file is relative to the keys
   directory specified in the +keysdir+ command or default
-  +/usr/local/etc+. Following are the subcommands:
+  +/etc+. Following are the subcommands:
 
   +cert+ _file_;;
     Specifies the location of the required host public certificate file.
@@ -64,7 +64,7 @@
 
 +keysdir+ _path_::
   This command specifies the default directory path for cryptographic
-  keys, parameters and certificates. The default is +/usr/local/etc/+.
+  keys, parameters and certificates. The default is +/etc+.
 
 [[trustedkey]]
 +trustedkey+ _key..._ ::
--- a/docs/includes/ntpd-body.txt
+++ b/docs/includes/ntpd-body.txt
@@ -511,7 +511,7 @@
 |log file           |system log      |+-l+        |+logfile+
 |include file       |none            |none        |+includefile+
 |statistics path    |+/var/NTP+      |+-s+        |+statsdir+
-|keys path          |+/usr/local/etc+|none        |+keysdir+
+|keys path          |+/etc+          |none        |+keysdir+
 |================================================================
 
 [[signals]]
--- a/pylib/packet.py
+++ b/pylib/packet.py
@@ -1418,7 +1418,7 @@
         "Retrieve ifstats data."
         return self.__ordlist("ifstats")
 
-DEFAULT_KEYFILE = "/usr/local/etc/ntp.keys"
+DEFAULT_KEYFILE = "/etc/ntp.keys"
 
 class Authenticator:
     "MAC authentication manager for NTP packets."
